scss/_custom-forms.scss

```diff
@@ -210,6 +210,8 @@
 .custom-file-input {
   min-width: $custom-file-width;
   max-width: 100%;
+  padding-top: $custom-file-padding-y;
+  padding-bottom: $custom-file-padding-y;
```

Comment:
My point was more that these 2 lines seem redundant due to these: https://github.com/twbs/bootstrap/pull/20033/files/3e5c6dcc07dc25dbb0e11411bef68a17ad8e9c2a#diff-e5540857a54676b43146ad92add2dbdeR231